src/main/java/ee/test_gov/sid/mock/config/SidMockProperties.java

```diff
@@ -10,10 +10,13 @@
 @ConfigurationProperties("sid-mock")
 public record SidMockProperties(
         @NotBlank
-        Duration delay, // = Duration.ofSeconds(1);
+        Duration delay,
 
         @NotBlank
-        Duration expiration // = Duration.ofMinutes(5);
+        Duration expiration,
+
+        @NotBlank
+        Boolean storeAuthRequests
```

Comment:
There needs to be some kind of documentation regarding these properties either here or in README.